<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FitnessTestScreen</class>
 <widget class="QWidget" name="FitnessTestScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget[objectName|=&quot;invisibleButton&quot;]{border: none; }</string>
  </property>
  <widget class="QLabel" name="backgroundLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>768</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/images/Fitness-Test---06---(Fitness---Standard).png</pixmap>
   </property>
  </widget>
  <widget class="InvisibleButton" name="invisibleButton_5">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>40</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="InvisibleButton" name="invisibleButton_6">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>640</y>
     <width>181</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="InvisibleButton" name="invisibleButton_fitness">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="InvisibleButton" name="invisibleButton_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>280</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="InvisibleButton" name="invisibleButton_3">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>280</y>
     <width>121</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="InvisibleButton" name="invisibleButton_4">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>160</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="InvisibleButton" name="invisibleButton_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>151</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="InvisibleButton" name="invisibleButton_8">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>191</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="maleSelectedLabel">
   <property name="geometry">
    <rect>
     <x>717</x>
     <y>523</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/images/RadioButton_Selected.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="femaleSelectedLabel">
   <property name="geometry">
    <rect>
     <x>849</x>
     <y>523</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/images/RadioButton_Selected.png</pixmap>
   </property>
  </widget>
  <widget class="InvisibleButton" name="invisibleButton_femaleRadio">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>490</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="InvisibleButton" name="invisibleButton_maleRadio">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>490</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
